packages/circuit-ui/components/NotificationBanner/__snapshots__/NotificationBanner.spec.tsx.snap

```diff
@@ -263,6 +264,7 @@ exports[`NotificationBanner styles should render a promotional banner 1`] = `
       alt="Update"
       class="circuit-8"
       src="/images/software_update.png"
+      width="100"
```

Comment:
Hm I didn't expect this but I think it's actually not such a bad idea, if I recall correctly it makes sense to pass a width and height to the img element to help the browser reduce layout shifts

Reply:
Yes, that's what I thought too. I verified that it doesn't affect the layout.